test/util.dart

```diff
@@ -24,19 +23,13 @@ Future<void> testDirectory(String name, {ExtensionSet? extensionSet}) async {
   }
 }
 
-Future<String> get markdownPackageRoot async {
```

Comment:
We can just assume tests are run from the root of the package. This vestigial logic is very much no longer needed.

Reply:
This may be causing test failures for the roll to google3.

We should revert this (or filter out related tests in google3).

Reply:
Oh. Ugh.